controllers/suite_test.go

```diff
@@ -17,26 +17,34 @@ limitations under the License.
 package controllers
```

Comment:
I think I'll likely revert the changes to this testing suite until I have an implementation that can easily mock catalog sourcing to avoid registering OLM's CatalogSource API here. Longer term, most of the e2e tests introduced in this PR can/should be modeled as unit tests.

Reply:
This is done in the latest force pushes.